app/views/hiring_staff/sign_in/email/sessions/choose_organisation.html.haml

```diff
@@ -9,10 +9,16 @@
 
       %p= t('hiring_staff.temp_login.choose_organisation.please_select')
 
-      - @schools.each do |school|
-        = link_to(location(school), auth_email_create_session_path(urn: school.urn), class: 'govuk-link')
+      - @local_authorities.each do |la|
```

Comment:
it is ok to have links, but a nicer radio buttons interface like DSI would be better here

Reply:
It would be but that's a piece of work that might never be used.